src/storage/src/hummock/sstable_store.rs

```diff
@@ -157,6 +161,10 @@ impl SstableStore {
         let block_cache_v2 = HybridCacheBuilder::new()
             .memory(block_cache_capacity)
             .with_shards(1)
+            .with_weighter(|_: &SstableBlockIndex, value: &Box<Block>| {
+                // FIXME(MrCroxx): Calculate block weight more accurately.
+                u64::BITS as usize * 2 / 8 + value.raw().len()
```

Comment:
shall it be `sizeof::<SstableBlockIndex>()` ?

Reply:
Yes. I manually calculate here to avoid anyone who refactors the code here in the future to use `sizeof` for the boxed value, too.

Reply:
I'll refactor it in another PR and leave a comment. Let's merge this first and unlock the main branch.